### ISSUE TITLE
Need an event/data point to know when a frame finished loading the about:blank document

### ISSUE THREAD

**Issue by OrKoN** [w3c/webdriver-bidi#832]
Initial about:blank loads are not navigations but clients often need to know if a frame is finishing loading the about:blank document. It is relevant for distinguishing regular frames from lazy loading frames that are paused and are not loading.

cc @sadym-chromium

**Comment by OrKoN** [comment]
(not sure if there is a duplicate spec issue for this)

**Comment by OrKoN** [comment]
> Is this request similar to the [`Page.ready` event](https://github.com/microsoft/playwright/blob/b0cec5b3518b8fa5d107068ea39612bcc1341b53/browser_patches/firefox/juggler/protocol/PageHandler.js#L88) that Playwright is using via Juggler with Firefox?

yes, I think it is. I think you mentioned this on some other issue but I could not find it.

**Comment by hbenl** [comment]
> It is relevant for distinguishing regular frames from lazy loading frames that are paused and are not loading.

Another important use case is pages opened by `window.open()` or `window.open(url)`: it's impossible for the client to tell from the `contextCreated` event if the page will navigate to a URL or if it will stay at `about:blank` (in which case no more events will be sent after `contextCreated`).

**Comment by whimboo** [comment]
> Another important use case is pages opened by `window.open()` or `window.open(url)`: it's impossible for the client to tell from the `contextCreated` event if the page will navigate to a URL or if it will stay at `about:blank` (in which case no more events will be sent after `contextCreated`).

Therefor see https://github.com/w3c/webdriver-bidi/issues/766 as well.
